string drone_id
string command
float32 posX
float32 posY
float32 posZ
